### PR TITLE
Update linux networking readme

### PR DESCRIPTION
**Pull request by nupuruttarwar**
Signed-off-by: Nupur Uttarwar <nupur.uttarwar@intel.com>

### CODE REVIEW

File: p4src/linux_networking/README_LINUX_NETWORKING.md [hunk 1]
```diff
@@ -37,17 +37,17 @@ This README describes a step by step procedure to run Linux networking scenario
 * Notes about topology:
   * VHOST ports, TAP ports, Physical LINK ports are created by GNMI CLI and LINK port is binded to the DPDK target
   * VLAN 1, VLAN 2, .... VLAN N created using Linux commands and are on top a TAP port. These VLAN ports should be equal to number of VM's that are spawned.
-  * br-int, VxLAN0 ports are created using ovs-vsctl command provided by P4-OVS and all the VLAN ports are attached to br-int using ovs-vsctl command.
+  * br-int, VxLAN0 ports are created using ovs-vsctl command provided by networking recipe and all the VLAN ports are attached to br-int using ovs-vsctl command.
```
Comment (ffoulkes) [inline p4src/linux_networking/README_LINUX_NETWORKING.md:40]: "Networking recipe", unlike "P4-OVS", takes a definite article ("the networking recipe")

File: p4src/linux_networking/README_LINUX_NETWORKING.md [hunk 2]
```diff
@@ -127,39 +126,51 @@ Note:
...
+
+sudo $IPDK_RECIPE/install/bin/ovs-vsctl add-br br-int
+ifconfig br-int up
 ```
-    ovs-vsctl add-port br-int vxlan1 -- set interface vxlan1 type=vxlan options:local_ip=40.1.1.1 options:remote_ip=40.1.1.2 options:dst_port=4789
-Note:
-    - VXLAN destination port should always be standard port. i.e., 4789. (limitation by p4 parser)
+
+#### 9. Configure VXLAN port
+   Using one of the TAP port for tunnel termination.
```
Comment (ffoulkes) [inline p4src/linux_networking/README_LINUX_NETWORKING.md:155]: "one of the TAP ports" (plural noun)

**Review by ffoulkes — APPROVED** [review]
I've noted a couple of grammar corrections.

File: p4src/linux_networking/README_LINUX_NETWORKING.md (hunk 1 above)
Comment (nupuruttarwar) [reply p4src/linux_networking/README_LINUX_NETWORKING.md:40]: Done

File: p4src/linux_networking/README_LINUX_NETWORKING.md (hunk 2 above)
Comment (nupuruttarwar) [reply p4src/linux_networking/README_LINUX_NETWORKING.md]: done

**Pull request merged**